### PR TITLE
Add checksum for py-tables 3.7.0

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
Confirmed sha256.

File: var/spack/repos/builtin/packages/py-tables/package.py [hunk 1]
```diff
@@ -25,6 +26,7 @@ class PyTables(PythonPackage):
     variant("lzo", default=False, description="Support for lzo compression")
 
     # requirements.txt
+    depends_on("python@3.6:", when="@3.7.0:", type=("build", "run"))
     depends_on("python@3.5:", when="@3.6.1:", type=("build", "run"))
     depends_on("python@3.4:", when="@3.6.0:", type=("build", "run"))
     depends_on("python@2.6:", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-tables/package.py:32]: Need deps from `pyproject.toml`:
```toml
requires = [
    "setuptools >=42.0",
    "wheel",
    "oldest-supported-numpy",
    "packaging",
    "Cython >=0.29.21",
]
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-tables/package.py (hunk 1 above)
Comment (iarspider) [reply var/spack/repos/builtin/packages/py-tables/package.py:32]: I will add py-oldest-supported-numpy later.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-tables/package.py:32]: That's not a real package, you don't need to add that.

File: var/spack/repos/builtin/packages/py-tables/package.py [hunk 2]
```diff
@@ -25,15 +26,19 @@ class PyTables(PythonPackage):
...
 
     depends_on("py-setuptools", type=("build", "run"))
+    depends_on("py-setuptools@42.0:", when="@3.7.0:", type=("build", "run"))
     depends_on("py-cython@0.21:", type="build")
+    depends_on("py-cython@0.29.21:", when="@3.7.0:", type="build")
     depends_on("py-numpy@1.9.3:", type=("build", "run"))
     depends_on("py-numexpr@2.6.2:", type=("build", "run"))
     depends_on("py-six@1.9.0:", when="@:3.5", type=("build", "run"))
+    depends_on("py-packaging", when="@3.7.0:", type="build")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-tables/package.py]:
```suggestion
    depends_on("py-packaging", when="@3.7.0:", type=("build", "run"))
```
Also included in `install_requires` and imported at run-time.

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-tables/package.py (hunk 1 above)
Comment (iarspider) [reply var/spack/repos/builtin/packages/py-tables/package.py:32]: @adamjstewart huh? https://pypi.org/project/oldest-supported-numpy/

Comment (iarspider) [reply var/spack/repos/builtin/packages/py-tables/package.py:32]: I  guess Spack handles that itself, right?

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**